packages/block-library/src/navigation-overlay-close/block.json

```diff
@@ -0,0 +1,49 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/navigation-overlay-close",
+	"title": "Navigation Overlay Close",
```

Comment:
Don't worry - we can make a separate PR for this block if/when we decide to merge this branch.

Reply:
@WordPress/interactivity-api This block is going to effectively be the same as the existing button which toggles the overlay. It's required because when editing the overlay within the editor, users need to be able to:

- perceive the presence of the Close button
- style the Close button directly

I'd really appreciate any advice/help on how best to transfer the interactivity directives over the this block 🙏

Reply:
We've been doing so with the `WP_HTML_Tag_Processor` quite successfully so far.

We basically use the `WP_HTML_Tag_Processor` on the render callback of the block, search for the inner block, and add the necessary directives. Something like this:

```php
$p = new WP_HTML_Tag_Processor( $block_content );

if ( $p->next_tag( array( 'class' => 'wp-block-button' ) ) ) {
  $p->set_attribute( 'data-wp-on--click', 'actions.openMenu' );
}
```